FirebaseCore/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Firebase 11.4.2
+- [fixed] CocoaPods only release to fix iOS 12 build failure resulting from
+  missing OS availability check in the `FirebaseCoreExtension` SDK.
```

Comment:
"incomplete implementation in the `FirebaseCoreInternal` CocoaPod."

Reply:
Done.